pegged/peg.d

```diff
@@ -1755,8 +1760,12 @@ template or(rules...) if (rules.length > 0)
     }
 }
 
-auto getUpto(ParseTree[] children, size_t minFailedLength) pure nothrow {
-    return children.filter!(r => max(r.end, r.failEnd) >= minFailedLength).array();
+auto getUpto(scope ParseTree[] children, size_t minFailedLength) pure nothrow {
+    auto ret = children.filter!(r => max(r.end, r.failEnd) >= minFailedLength).array();
+
+    // FIXME: .array() garantees a copy or the elements. Phobos/compiler scope
```

Comment:
"or the elements" should be "of the elements", I presume.

Reply:
Done